### PR TITLE
Support macOS and tvOS in GDT and GDTCCT

### PR DESCRIPTION
**Pull request by mikehaney24**

### CODE REVIEW

**Comment by mikehaney24** [comment]
The GoogleDataTransportCCTSupport travis won't pass until I push the GoogleDataTransport podspec to SpecStaging.

File: GoogleDataTransport/GDTLibrary/GDTPlatform.m [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <GoogleDataTransport/GDTPlatform.h>
+
+#if TARGET_OS_IOS || TARGET_OS_TVOS
+const GDTBackgroundIdentifier GDTBackgroundIdentifierInvalid = 0;
+#else
+const GDTBackgroundIdentifier GDTBackgroundIdentifierInvalid = 0;
```
Comment (paulb777) [inline GoogleDataTransport/GDTLibrary/GDTPlatform.m]: These look the same.

Comment (mikehaney24) [reply GoogleDataTransport/GDTLibrary/GDTPlatform.m]: They are! Fixed.

**Comment by mikehaney24** [comment]
Confirmed, GDT CCT travis passes now, since I pushed to SpecsStaging

https://travis-ci.org/firebase/firebase-ios-sdk/jobs/535072527

**Pull request merged**